.gitignore

```diff
@@ -0,0 +1,25 @@
+#Maven
+
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+pom.xml.next
+release.properties
+dependency-reduced-pom.xml
+buildNumber.properties
+.mvn/timing.properties
+
+<<<<<<< HEAD
+
+#.iml
+Info-Klausur.iml
+
+#.idea
+.idea
+=======
```

Comment:
Das hier sind die Überreste eines [Merge Conflicts](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/). Der Block über `===` bis zum `<<<` war auf deinem Branch und der darunter bis zum `>>>` kam von `pomxml` (ich glaube das war sorum, vergesse das immer) - Git konnte sie nicht automatisch zusammenführen, also musst du das manuell machen. Git müsste dich dabei auch hingewiesen haben.

Du kannst dir entweder einen der beiden Blöcke aussuchen oder ein Mischmasch erstellen. In jedem Fall sollte nachher jede Zeile nur einmal da sein und die `<<<`, `===`, `>>>` Zeilen sollten raus sein.